### DISCUSSION TITLE
Molecule-level 2D features

### DISCUSSION THREAD

**Discussion by diegoluna3** [chemprop/chemprop#849] [Q&A]
Hello, in chemprop v1.7 there was support for [molecule-level rdkit 2d features](https://github.com/chemprop/chemprop/tree/v1.7.1?tab=readme-ov-file#molecule-level-rdkit-2d-features). I see they are briefly mentioned in the [new chemprop v2.0 docs](https://chemprop.readthedocs.io/en/main/tutorial/cli/train.html#additional-features) but I do not see support for them in the [molecule-level 2d features section](https://chemprop.readthedocs.io/en/main/tutorial/cli/train.html#molecule-level-2d-features). Are they no longer supported?

**Comment by kevingreenman** [comment]
We will provide an answer to this in #843 and have added this to out list of example notebooks to make (see #772)
